monkey/monkey_island/cc/services/config_schema/ransomware.py

```diff
@@ -2,32 +2,45 @@
     "title": "Ransomware",
     "type": "object",
     "properties": {
-        "directories": {
-            "title": "Directories to encrypt",
+        "encryption": {
+            "title": "Encryption",
             "type": "object",
             "properties": {
-                "linux_dir": {
-                    "title": "Linux encryptable directory",
-                    "type": "string",
-                    "default": "",
-                    "description": "Files in the specified directory will be encrypted "
-                    "using bitflip to simulate ransomware.",
+                "should_encrypt": {
+                    "title": "Encrypt files",
+                    "type": "boolean",
+                    "default": True,
+                    "description": "Selected files will be encrypted using bitflip to simulate "
+                    "ransomware. Enter target directories below.",
                 },
-                "windows_dir": {
-                    "title": "Windows encryptable directory",
-                    "type": "string",
-                    "default": "",
-                    "description": "Files in the specified directory will be encrypted "
-                    "using bitflip to simulate ransomware.",
+                "directories": {
+                    "title": "Directories to encrypt",
+                    "type": "object",
+                    "properties": {
+                        "linux_dir": {
+                            "title": "Linux encryptable directory",
+                            "type": "string",
+                            "default": "",
+                            "description": "Files in the specified directory will be encrypted "
+                            "using bitflip to simulate ransomware.",
+                        },
+                        "windows_dir": {
+                            "title": "Windows encryptable directory",
+                            "type": "string",
+                            "default": "",
+                            "description": "Files in the specified directory will be encrypted "
+                            "using bitflip to simulate ransomware.",
```

Comment:
"Ransomware encryption will be simulated by flipping every bit in the files contained within the specified directory"

Reply:
That seems repetitive to me. I modified it to: "Files in the specified directory will be encrypted using bitflip. If no directory is specified, no files will be encrypted."

Reply:
I think "encrypted using bitflip" is awkwardly worded. "Bitflip" isn't an established encryption mechanism like AES.

Reply:
I settled on, "A path to a directory on Windows systems that can be used to safely simulate the encryption behavior of ransomware. If no directory is specified, no files will be encrypted." We can always change it later.